src/scythe-bidder/board.tsx

```diff
@@ -203,13 +203,25 @@ const BiddingBoard = (props: {
                     "&:not(:first-child)": { marginTop: 12 },
                   }}
                 >
-                  {
+                  {c.faction === "Albion" ? (
+                    <img
+                      src={require(`./static/images/${c.faction}.webp`)}
+                      css={{ width: 24, height: 24, marginRight: 8 }}
+                      alt={c.faction}
+                    />
+                  ) : c.faction === "Togawa" ? (
+                    <img
+                      src={require(`./static/images/${c.faction}.webp`)}
+                      css={{ width: 24, height: 24, marginRight: 8 }}
+                      alt={c.faction}
+                    />
+                  ) : (
```

Comment:
Why do we have the different cases when the inner code is identical?

Reply:
Good point, changing that now!